etl/steps/data/garden/emissions/2024-04-08/national_contributions.countries.json

```diff
@@ -219,13 +217,11 @@
   "Yemen": "Yemen",
   "Zambia": "Zambia",
   "Zimbabwe": "Zimbabwe",
-  "LDC": "Least developed countries (Jones et al. 2023)",
-  "OECD": "OECD (Jones et al. 2023)",
-  "French Equatorial Africa": "French Equatorial Africa (Jones et al. 2023)",
-  "French West Africa": "French West Africa (Jones et al. 2023)",
-  "Kuwaiti Oil Fires": "Kuwaiti Oil Fires (Jones et al. 2023)",
-  "Leeward Islands": "Leeward Islands (Jones et al. 2023)",
-  "Panama Canal Zone": "Panama Canal Zone (Jones et al. 2023)",
-  "Ryukyu Islands": "Ryukyu Islands (Jones et al. 2023)",
-  "St. Kitts-Nevis-Anguilla": "St. Kitts-Nevis-Anguilla (Jones et al. 2023)"
+  "Kuwaiti Oil Fires": "Kuwaiti Oil Fires",
+  "Leeward Islands": "Leeward Islands",
+  "Panama Canal Zone": "Panama Canal Zone",
+  "Ryukyu Islands": "Ryukyu Islands",
+  "St. Kitts-Nevis-Anguilla": "St. Kitts-Nevis-Anguilla",
+  "LDC": "Least developed countries (Jones et al.)",
+  "OECD": "OECD (Jones et al.)"
 }
```

Comment:
There was overlapping data for "Palau" and for "Pacific Islands (Palau)", so I simply excluded this region instead of trying to figure out a better solution.